src/main/kotlin/dev/schlaubi/votebot/util/Colors.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Votebot - A feature-rich bot to create votes on Discord guilds.
+ *
+ * Copyright (C) 2019-2021  Michael Rittmeister & Yannick Seeger
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+package dev.schlaubi.votebot.util
+
+import dev.kord.common.Color
+
+/**
+ * Wrapper for [Discordapp.com/branding][https://discordapp.com/branding] colors and some other colors:
+ */
+@Suppress("KDocMissingDocumentation", "unused", "MagicNumber")
+object Colors {
+    // Discord
+    val BLURLPLE: Color = Color(114, 137, 218)
```

Comment:
Those are actually wrong update them to match the new branding

Reply:
Should I remove all the non-existing colors?

Reply:
Or e.g. replace the current LIGHT_GREEN with the branding GREEN

Reply:
@DRSchlaubi